person.rb

```diff
@@ -2,13 +2,17 @@
 
 class Person
   attr_writer :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
+
+  has_many :rentals
+  has_many :books, through: :rentals
```

Comment:
- [ ] `has_many` are not valid methods. Kindly remove them as they return **`NoMethodErrors`** when creating an object